compiler/main/version.cpp

```diff
@@ -68,3 +76,38 @@ get_clang_sysroot_args() {
   expandInstallationPaths(ret);
   return ret;
 }
+
+void makeVersionModule(const ArgumentDescription *arg, const char* str) {
+  printf("module ChplVersion {\n");
+  printf("/* A ``string`` representing the version of `chpl` */\n");
+  printf("  param CHPL_VERSION: string = \"%d.%s.%s",
+         MAJOR_VERSION, MINOR_VERSION, UPDATE_VERSION);
+  if (!official) {
+    printf(" pre-release (%s)", BUILD_VERSION);
+  }
+  printf("\";\n");
+  printf("/* An ``int`` representing the major version number of `chpl` */\n");
+  printf("  param CHPL_VERSION_MAJOR: int = %d;\n", MAJOR_VERSION);
+  printf("/* An ``int`` representing the minor version number of `chpl` */\n");
+  printf("  param CHPL_VERSION_MINOR: int = %s;\n", MINOR_VERSION);
+  printf("/* An ``int`` representing the update version number of `chpl` */\n");
+  printf("  param CHPL_VERSION_UPDATE: int = %s;\n", UPDATE_VERSION);
+  printf("/* A ``bool`` indicating whether or not this is an official release of `chpl` */\n");
+  printf("  param CHPL_VERSION_OFFICIAL: bool = ");
```

Comment:
I'd probably call this CHPL_VERSION_IS_RELEASE.

Reply:
Based on the other feedback on issue #5491, I ended up changing it to `versionIsRelease`.